recipe/meta.yaml

```
@@ -21,11 +21,12 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - hatchling 1.12.2
```

```suggestion
    - hatchling >=1.6.0,<2
```

As it is a python build tool, I would leave the constraint more open.
This makes rebuild easier.